transport/tchannel/channel_outbound.go

```diff
@@ -1,96 +1,71 @@
-// Copyright (c) 2016 Uber Technologies, Inc.
-//
-// Permission is hereby granted, free of charge, to any person obtaining a copy
-// of this software and associated documentation files (the "Software"), to deal
-// in the Software without restriction, including without limitation the rights
-// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-// copies of the Software, and to permit persons to whom the Software is
-// furnished to do so, subject to the following conditions:
-//
-// The above copyright notice and this permission notice shall be included in
-// all copies or substantial portions of the Software.
-//
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-// THE SOFTWARE.
-
 package tchannel
 
 import (
 	"context"
 	"io"
 
+	"github.com/uber/tchannel-go"
+	"go.uber.org/atomic"
 	"go.uber.org/yarpc/internal/encoding"
 	"go.uber.org/yarpc/internal/errors"
 	"go.uber.org/yarpc/transport"
-
-	"github.com/opentracing/opentracing-go"
-	"github.com/uber/tchannel-go"
-	"go.uber.org/atomic"
 )
 
-var errOutboundNotStarted = errors.ErrOutboundNotStarted("tchannel.Outbound")
-
-// NewOutbound builds a new TChannel outbound which uses the given Channel to
-// make requests.
-func NewOutbound(ch Channel) *Outbound {
-	return &Outbound{channel: ch, started: atomic.NewBool(false)}
+// NewOutbound builds a new TChannel outbound using the transport's shared
+// channel to make requests to any connected peer.
+func (t *ChannelTransport) NewOutbound() *ChannelOutbound {
+	return &ChannelOutbound{
+		started:   atomic.NewBool(false),
```

Comment:
we don't need to set these every time, we can initialize `started` as a atomic.Bool and it will automatically be set to false.